Doc/glossary.rst

```diff
@@ -388,19 +390,21 @@ Glossary
       and the :ref:`function` section.
 
    function annotation
-      An :term:`annotation` of a function, or a method.
+      An :term:`annotation` of a function, or a method parameters and return
```

Comment:
I'd say "An annotation of a function parameter or return type." (plus markup on "annotation")